### ISSUE TITLE
Override tab width using `tabWidth` option

### ISSUE THREAD

**Issue by bminer** [jonlabelle/SublimeJsPrettier#143]
### Explain the Issue and Expected Behavior

Current Behavior: `tabWidth` option does not override Sublime `tab_size`
Expected Behavior: Setting `prettier_options.tabWidth` should override Sublime's `tab_size`.

Relevant part of program (this should instead be changed to check if `tabSize` prettier option was explicitly overwritten): https://github.com/jonlabelle/SublimeJsPrettier/blob/7cbe4e5e39148fe5f42b565ce69ee7bfcf48c9f6/JsPrettier.py#L527

### Prettier version

1.14.3

### JsPrettier Plug-in Version

1.21.1

### Platform Details

Provide your Sublime Text version and Platform details.

    - Sublime Text Version: 3.0
    - Sublime Text Build: 3143
    - Sublime Text Architecture: x86_64
    - Operating System Name: Linux Mint
    - Operating System Version: 18.3
    - Operating System Architecture: x86_64

### Generated Prettier command line arguments

To view the generated prettier command line arguments you need to enable JsPrettier's [debug setting] and open the Sublime Text Console after a file/section formatting attempt.

```
....../node_modules/.bin/prettier --stdin --config ..../.prettierrc --config-precedence cli-override --parser babylon --tab-width 4 --use-tabs true --stdin-filepath ....../History.js --cursor-offset 696
```

Note: Above path info has been omitted for brevity.  Also note that despite having set `prettier_options.tabWidth` to the value 2, the `--tab-width 4` option was passed as a CLI arg.

### Is the same behavior observed when run against Prettier directly?

N/A

### The contents of your `User/JsPrettier.sublime-settings` file

It's empty
    
### The contents of your `<project_name>.sublime-project` file (if applicable)

```json
{
	"folders":
	[
		{
			"path": ..., // omitted
			"folder_exclude_patterns": ["node_modules", ".cache", "frontend/dist", "vendor"]
		}
	],
	"settings":
	{
		"ensure_newline_at_eof_on_save": true,
		"js_prettier": {
			"auto_format_on_save": true,
			"auto_format_on_save_excludes": [],
			"auto_format_on_save_requires_prettier_config": true,
			"prettier_cli_path": ..., // omitted for brevity
			"debug": true,
			"allow_inline_formatting": false,
			"custom_file_extensions": [],
			"max_file_size_limit": -1,
			"prettier_options": {
				"tabWidth": 2 // the part that's broke
			}
			// "additional_cli_args": {
			// 	"--tab-width": 2
			// } // Note: tried this and the plugin passes it along, but prettier CLI does not like it
		}
	}
}
```

### Steps to reproduce the behavior

1. Try to override `tabWidth` behavior
2. Realize it doesn't work

**Comment by jonlabelle** [comment]
This is the intended behavior, and to adhere to what's *already* configured in SublimeText. Otherwise, if you specify a Prettier `tabWidth` that doesn't align with what's configured for `tab_size` (in SublimeText), you're likely to get mixed results... and more so if you're formatting inline code.

**Comment by bminer** [comment]
@jonlabelle - Sorry for bothering you with this.  I am grateful to have Sublime + JSPrettier integration, but I'm still having an issue forcing a particular tab width.  Different developers may have different tab widths, but you still may want Prettier to produce consistent results; otherwise, committing a file in a Git repo would result in back and forth with the tab widths, producing nonsensical diffs.

Basically, I would like there to be a way to force a tab width, regardless of the Sublime Text `tab_width` setting.  What do you think?

**Comment by bminer** [comment]
FYI - If you feel like it would be appropriate to re-open this issue, I'd be happy to provide a PR.  Thanks.

**Comment by jonlabelle** [comment]
Approved. I will intro it shortly.

**Comment by bminer** [comment]
@jonlabelle - Thanks so much!
